### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by jamesmstone** [ryan4yin/nixos-and-flakes-book#152] [General]
Just wanted to say thanks. Your book has been the key resource that made nix click for me. Thanks. Can I support you and the book in some way?

**Comment by jamesmstone** [comment]
found this, thanks 
https://buymeacoffee.com/ryan4yin/c/9249960

**Reply by ryan4yin** [reply]
Glad to help. Thanks for your sponsorship!
